dapps/react-dapp-v2/src/contexts/JsonRpcContext.tsx

```diff
@@ -814,6 +830,169 @@ export function JsonRpcContextProvider({
     ),
   };
 
+  // -------- ELROND RPC METHODS --------
+
+  const elrondRpc = {
+    testSignTransaction: _createJsonRpcRequestHandler(
+      async (
+        chainId: string,
+        address: string
+      ): Promise<IFormattedRpcResponse> => {
+        const reference = chainId.split(":")[1];
+        const sender = new Address(address);
+        const testTransaction = new ElrondTransaction({
+          nonce: 1,
+          value: "10000000000000000000",
+          receiver: Address.fromBech32(address),
+          sender,
+          gasPrice: 1000000000,
+          gasLimit: 50000,
+          chainID: reference,
+          data: new TransactionPayload("testdata"),
+        });
+        const transaction = testTransaction.toPlainObject();
+
+        try {
+          const result = await client!.request<{ signature: ISignature }>({
+            chainId,
+            topic: session!.topic,
+            request: {
+              method: DEFAULT_ELROND_METHODS.ELROND_SIGN_TRANSACTION,
+              params: {
+                transaction,
+              },
+            },
+          });
+
+          const valid = true; // Implement transaction signature validity check once it is implemented in erdjs
```

Comment:
Is this still current, i.e. is this available now as part of the `erdjs` lib?

If not is there another, more low-level way to validate the returned signatures instead of defaulting to `true`?

Reply:
Added the validity check for Elrond transacions too